e2e_tests/tests/task/test_generic_tasks.py

```diff
@@ -0,0 +1,200 @@
+import pathlib
+import subprocess
+
+import pytest
+
+from determined.cli import ntsc
+from determined.common import context, util
+from determined.common.api import bindings
+from tests import api_utils
+from tests import config as conf
+
+
+@pytest.mark.e2e_cpu
+def test_create_generic_task() -> None:
+    """
+    Start a simple task with a context directory called from the task CLI
+    """
+    command = [
+        "det",
+        "-m",
+        conf.make_master_url(),
+        "task",
+        "create",
+        conf.fixtures_path("generic_task/test_config.yaml"),
+        "--context",
+        conf.fixtures_path("generic_task"),
+    ]
+
+    subprocess.run(command, universal_newlines=True, stdout=subprocess.PIPE, check=True)
+
+
+@pytest.mark.e2e_cpu
+def test_generic_task_config() -> None:
+    """
+    Start a simple task without a context directory and grab its config
+    """
+    test_session = api_utils.determined_test_session()
+
+    with open(conf.fixtures_path("generic_task/test_config.yaml"), "r") as config_file:
+        # Create task
+        config_text = config_file.read()
```

Comment:
Two things here:

- don't indent the entire test as part of this `with`.  You only need `config_file` active long enough for this call to `config_file.read()`
- why even bother with a fixture for this test?  Fixtures are annoying because they spread the test logic over multiple files.  If you can just do `config_file = 'entrypoint: ["echo", "hi"]'` then the test is easier to read
- Also such a change makes it so you don't need to call `read_v1_context` either

Reply:
Hey, you marked this as resolved but didn't take any action.

Please don't do that.

The fixtures have been discussed at length, but the part about indenting everything unnecessarily still stands.